config

```diff
@@ -102,92 +102,41 @@ END
    exit 1
 fi
 
-ngx_version=`grep nginx_version src/core/nginx.h | sed -e 's/^.* \(.*\)$/\1/'`
-
-if [ -z "$ngx_version" ]; then
-   cat << END
- $0: error: ngx_postgres addon was unable to detect version of nginx.
-END
-   exit 1
-fi
-
-# work-around for versions of nginx older than nginx-0.9.0
-if [ $ngx_version -ge 9000 ]; then
-    ngx_feature_name="NGX_POSTGRES_LIBRARY_VERSION"
-    ngx_feature_run=value
-else
-    ngx_feature_name="NGX_POSTGRES_LIBRARY_VERSION_DETECTED"
-    ngx_feature_run=no
-fi
-
-lib_version=90100
-ngx_feature="libpq library version 9.1"
-ngx_feature_test="printf(\"%d\", PQlibVersion())"
-. auto/feature
-
-if [ $ngx_found = no ]; then
-    lib_version=90000
-    ngx_feature="libpq library version 9.0"
-    ngx_feature_test="(void) PQescapeLiteral(NULL, NULL, 0);
-                      printf(\"$lib_version\")"
-    . auto/feature
-fi
```

Comment:
Will you please explain these changes. I prefer minimal changes.

Reply:
Does everything else look good?